packages/repository/README.md

```diff
@@ -8,9 +8,9 @@ refactored and decomposed into multiple modules as we refine the story based on
 the legacy `loopback-datasource-juggler` and connector modules from LoopBack
 3.x.
 
-# Concepts
+## Overview
```

Comment:
I think this top section needs a bit of rework. Here is what our guidelines are saying:

> Summary
>
> A one-sentence description of what the module does. This should be the same as the npmjs.org blurb (which comes from the description property of package.json). Since npm doesn’t handle markdown for the blurb, avoid using markdown in the summary sentence.
>
> Overview
>
> A paragraph with more high-level information on what the module does, what problems it solves, why one would use it and how, etc. For larger modules, this would include a bullet-list of features.

I think the contents on L3-L9 belongs to Overview, we need to add a one-sentence Summary. Ideally, the same sentence should be used in `package.json` as `description` field.